lib/app.js

```diff
@@ -200,13 +200,18 @@ function App () {
 
   initSSE(app)
 
+  // Data integration experimental endpoints created by James McLaughlin
   if (config.get('experimental').dataIntegration) {
     app.get('/jobs', requireUser, views.jobs)
     app.post('/actions/job/cancel', requireUser, actions.cancelJob)
     app.post('/actions/job/restart', requireUser, actions.restartJob)
     app.get('/admin/jobs', requireAdmin, views.admin.jobs)
     app.all('/user/:userId/:collectionId/:displayId/:version([^\\.]+)/integrate', requireUser, views.dataIntegration)
     app.all('/public/:collectionId/:displayId/:version([^\\.]+)/integrate', views.dataIntegration)
+    app.get('/user/:userId/:collectionId/:displayId/:version/createImplementation', requireUser, actions.createImplementation)
+    app.get('/user/:userId/:collectionId/:displayId/:version/createTest', requireUser, actions.createTest)
+    // TODO: need to decide if createSnapshot is functional and should be kept or not
+    app.get('/public/:collectionId/:displayId/:version/createSnapshot', actions.createSnapshot)
```

Comment:
How come these are merged into the dataIntegration section?

Reply:
I put them there as they are experimental and likely in most cases not actively developed.  I think createTest and createImplementation really never got quite finished and don't use the new SBOL classes.  I'm not sure the status of createSnapshot.  Thought it might be best to leave them in until we decide what to do with them, but not make them exposed in the release version.

Reply:
Ok, fine by me